From c783e7e3f2ff7adeed7779913020f2c9dcda2ef9 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Thu, 11 Oct 2018 18:33:15 +0300
Subject: [PATCH 322/706] arm64: dts: ls1028a: Drop phy nodes from CFP dts to
 speed up i/f open

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit 648c9e0e70d2438b46f2ab4ca7426e00a3d37d07)
(cherry picked from commit 69f2f98bb44f3b87241e8f92f199e01843c524c7)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../boot/dts/freescale/fsl-ls1028a-cfp.dts    | 32 -------------------
 1 file changed, 32 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-cfp.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-cfp.dts
index 604a11c639c2..399ba6012d03 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-cfp.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-cfp.dts
@@ -56,35 +56,3 @@
 		crypto = &crypto;
 	};
 };
-
-&enetc_port0 {
-	phy-connection-type = "internal";
-	fixed-link {
-		speed = <1000>;
-		full-duplex;
-	};
-};
-
-&enetc_port1 {
-	phy-connection-type = "internal";
-	fixed-link {
-		speed = <1000>;
-		full-duplex;
-	};
-};
-
-&enetc_port2 {
-	phy-connection-type = "internal";
-	fixed-link {
-		speed = <1000>;
-		full-duplex;
-	};
-};
-
-&enetc_port3 {
-	phy-connection-type = "internal";
-	fixed-link {
-		speed = <1000>;
-		full-duplex;
-	};
-};
-- 
2.17.1

